client/src/components/Login/index.js

```diff
@@ -8,113 +8,126 @@ import { LOGIN_USER } from '../../utils/mutations';
 import Auth from '../../utils/auth';
 
 function Login() {
-	const [isSignOpen, setIsSignOpen] = useState(false);
+  const [isSignOpen, setIsSignOpen] = useState(false);
+  let navigate = useNavigate();
+  const toggleSign = () => {
+    setIsSignOpen(!isSignOpen);
+  };
 
-	const toggleSign = () => {
-		setIsSignOpen(!isSignOpen);
-	};
+  const [formState, setFormState] = useState({ email: '', password: '' });
+  const [login, { error }] = useMutation(LOGIN_USER);
 
-	const [formState, setFormState] = useState({ email: '', password: '' });
-	const [login, { error }] = useMutation(LOGIN_USER);
+  // update state based on form input changes
+  const handleChange = (event) => {
+    const { name, value } = event.target;
 
-	// update state based on form input changes
-	const handleChange = (event) => {
-		const { name, value } = event.target;
+    setFormState({
+      ...formState,
+      [name]: value,
+    });
+  };
 
-		setFormState({
-			...formState,
-			[name]: value,
-		});
-	};
+  // submit form
+  const handleFormSubmit = async (event) => {
+    event.preventDefault();
+    console.log('handleFormSubmit login');
+    try {
+      console.log('==============logging in===================');
```

Comment:
You should remove the console.log used for testing prior to a pushing your commit.